### DISCUSSION TITLE
Global Toast Service Customisation

### DISCUSSION THREAD

**Discussion by heather-timmins** [vikramlearning/blazorbootstrap#665] [Q&A]
How can I customise the Global Toast Service? 
I can't seem to access the variables the same as page level toasts? 
(eg. Including a title / body / etc) 

Using the below code: 

<Toasts class="p-3" AutoHide="true" Delay="4000" Placement="ToastsPlacement.TopRight" />

@code {

    [Inject] protected ToastService ToastService { get; set; } = default!;

    private void SaveEmployee()
    {
        try
        {
            // TODO: call the service/api to save the employee details

            ToastService.Notify(new(ToastType.Success, $"Employee details saved successfully."));
        }
        catch(Exception ex)
        {
            // handle exception

            ToastService.Notify(new(ToastType.Danger, $"Error: {ex.Message}."));
        }
    }
}

**Comment by gvreddy04** [comment]
@heather-timmins Can you elaborate on your scenario for customization with some examples? This will help us understand it in a better way.

**Reply by heather-timmins** [reply]
@gvreddy04  Hi, Yes: 

**This is the toast style that shows using the global toast service:** 
![image](https://github.com/vikramlearning/blazorbootstrap/assets/144691290/50e75ba7-4839-4836-b667-1c6153fb6a90)
**Using this code:**


```
<Toasts class="p-3" AutoHide="true" Delay="4000" Placement="ToastsPlacement.BottomLeft" />
code {
[Inject] protected ToastService ToastService { get; set; } = default!;
private void SaveEmployee()
{
    try
    {

        ToastService.Notify(new(ToastType.Success, $"Employee details saved successfully."));
    }
    catch(Exception ex)
    {

        ToastService.Notify(new(ToastType.Danger, $"Error: {ex.Message}."));
    }
}
}
```

**This is the toast style I want to display on the global service :** 
![image](https://github.com/vikramlearning/blazorbootstrap/assets/144691290/27b61ec0-3c44-49f4-b217-8e84a32dcbac)

**Reply by gvreddy04** [reply]
@heather-timmins Please create an issue for this. We will implement an enhancement.
